config.rb

```diff
@@ -14,9 +14,8 @@
 
 # (Semi-) secrets
 set :segmentio_writekey, ENV['SEGMENTIO_WRITEKEY'] || 'cn8oifbk6o'
-set :swiftype_key, ENV['SWIFTYPE_KEY'] || 'dsMEc1fYviE2ShXAjYMW'
-set :swiftype_engine, ENV['SWIFTYPE_ENGINE'] || 'axuhZ5Lt1ZUziN-DqxnR'
 set :base_url, ENV['BASE_URL'] || 'https://www.aptible.com'
+set :base_doc_url, ENV['BASE_DOC_URL'] || 'https://www.aptible.com'
```

Comment:
Should we overwrite `BASE_DOC_URL` for staging in `Rakefile`, where we also override `BASE_URL`?